e2e_playwright/st_button.py

```diff
@@ -105,6 +105,8 @@ def on_click(x: int, y: int) -> None:
     st.button("Stretch Width", width="stretch")
     st.button("200px Width", width=200)
 
+st.markdown("Dynamic button props:")
```

Comment:
The test below is flaky because the sizing is on the edge of a subpixel. One trick to resolve this is to add another markdown element -> which itself usually has a subpixel height